lib/recurly/export_file_list.php

```diff
@@ -11,7 +11,9 @@ class Recurly_ExportFileList extends Recurly_Pager
    * @return Recurly_ExportFileList
    */
   public static function get($date, $params = null, $client = null) {
-    return new self(self::_uriWithParams('/export_dates/' . rawurlencode($date) . '/export_files', $params), $client);
+    return new self(self::_uriWithParams(
+      self::_safeUri('/export_dates', $date, '/export_files'), $params
```

Comment:
Shouldn't none of these keys start with `/`? This is either adding double slashes or it's encoding the second slash right?

Reply:
It's encoding the second slash. We can avoid this by removing the preceding `/` in the PATH variables in `client.php` or replacing `%2F` with blanks. I opted to do the latter. 

In a case like this where we're not passing in a variable, I could instead pass in `export_files` instead of `/export_files`, for example.